### ISSUE TITLE
Call-to-action entry produces accessibility issue

### ISSUE THREAD

**Issue by philippfromme** [camunda/improved-canvas#43]
The new call-to-action entry is a `<div>` element but has a `aria-label` attribute and no `role` attribute which produces an accessibility  error.

![image](https://github.com/camunda/improved-canvas/assets/7633572/6da49447-8b99-4afc-a864-f9642cee5267)

**Comment by philippfromme** [comment]
Closed by https://github.com/camunda/improved-canvas/pull/44

**Comment by philippfromme** [comment]
This issue has not been fixed. `aria-role` doesn't exist. `role` should be used instead.
